examples/seq2seq/run_seq2seq.py

```diff
@@ -167,9 +167,22 @@ class DataTrainingArguments:
             "value if set."
         },
     )
+    max_test_samples: Optional[int] = field(
+        default=None,
+        metadata={
+            "help": "For debugging purposes or quicker training, truncate the number of test examples to this "
+            "value if set."
+        },
+    )
     source_lang: Optional[str] = field(default=None, metadata={"help": "Source language id for translation."})
     target_lang: Optional[str] = field(default=None, metadata={"help": "Target language id for translation."})
-    eval_beams: Optional[int] = field(default=None, metadata={"help": "Number of beams to use for evaluation."})
+    eval_beams: Optional[int] = field(
```

Comment:
It's `num_beams` everywhere in the core, so perhaps while we are re-shuffling things we could match that name? 

It's always eval when beam search is used, so it can't be train_beams, so `eval` is a redundant info in the current name.

Reply:
`num_beams` seems good to me. In the previous script, we called it `eval_beams` because we had two beams arguments one for eval and one for the test. But here we just have one, so `num_beams` makes sense to me. Let's wait for @sgugger  opinion.

Reply:
I have no strong opinion. It's another name variable change from the old script but we already have a few renames. If you think it's better as `num_beams`, let's go for the change!